core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.xul

```diff
@@ -44,58 +52,65 @@
               <treechildren></treechildren>
             </treeitem>
           </treechildren>
-
         </tree>
-
       </editpanel>
 
       <deck id="modelDeck" flex="1" height="465">
-
         <tabbox id="modelTabbox" flex="1" width="410">
           <tabs>
-            <tab id="analysisTab" label="${analysisTabLabel}" onclick="modeler.setModelerPerspective('ANALYSIS')"></tab>
+            <tab id="analysisTab" label="${analysisTabLabel}"
+                 onclick="modeler.setModelerPerspective('ANALYSIS')"></tab>
             <tab id="reportingTab" label="${reportingTabLabel}"
                  onclick="modeler.setModelerPerspective('REPORTING')"></tab>
           </tabs>
+
           <tabpanels>
             <tabpanel id="analysisTabPanel" height="409">
-
-              <vbox id="analyzerModelPanel" height="411" flex="1" spacing="0" padding="0">
-                <toolbar id="schema-toolbar" mode="icons">
+              <vbox id="analyzerModelPanel" height="411" flex="1"
+                    spacing="0" padding="0" pen:classname="model-deck-panel">
```

Comment:
I added this class to avoid specifying the two `vbox` ids that are `***ModelPanel`.
But now that I'm reading the class name and the ids, I see that it should be only `model-panel` 😅 

Regardless, I feel like we could simply use the ids because the rule in `datasourceEditorDialog.css` is not generic,
as we are just overriding the `height` with an `!important`